metadata_service/api/column.py

```diff
@@ -21,16 +21,18 @@ def __init__(self) -> None:
 
     def put(self,
             table_uri: str,
-            column_name: str,
-            description_val: str) -> Iterable[Union[dict, tuple, int, None]]:
+            column_name: str) -> Iterable[Union[dict, tuple, int, None]]:
         """
-        Updates column description
+        Updates column description (passed as a request body)
+        :param table_uri:
+        :param column_name:
+        :return:
         """
         try:
+            description = self.parser.parse_args()['description']
```

Comment:
@verdan , I assume it won't affect the atlas backend?

Reply:
It shouldn't. go ahead!!